time_stamper_cv/msg/Ledstate.msg

```diff
@@ -0,0 +1,5 @@
+Header header
+bool is_valid
+int32 counter
+int32[] binary_state
+uint8[] intensity
```

Comment:
as intensity is an averaged value of many uint8's (has more resolution than just a single uint8), you could also use a float/double representation here.